.github/workflows/soak-py.yml

```diff
@@ -0,0 +1,153 @@
+name: soak-python
+
+env:
+  LAYER: py38-layer
+  LAYER_PATH: 'opentelemetry-lambda/python/src'
+  STACK: py38-soak
+  UTILS: ${{ github.workspace }}/opentelemetry-lambda/utils/sam
+
+on:
+  workflow_dispatch:
+    inputs:
+      runtime:
+        description: 'soaking'
+        required: false
+        default: 'Python3.8'
+      args:
+        description: 'soaking test args'
```

Comment:
Maybe explain a bit more what the arguments actually do. Or better yet, make them a separate input for each configurable argument.